### PR TITLE
fix: hide non-static columns when comparison view open

### PR DESCRIPTION
**Pull request by EmilyBonar**
## Description
Only static columns (checkboxes and names) should be shown when the comparison view is open on the new experiment list page.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by keita-determined — APPROVED** [review]
the code looks good.
Do we want to hide non-static cols when comparison view is open? The width is adjustable, so i personally thought its nice to show non-static cols. 
I'll defer to @alexjdetermined for design decision. 

image

**Comment by alexjdetermined** [comment]
Issue
* "Sidebar" should show the frozen column(s) full width within the "sidebar". Currently there is what appears to be "another column" at after the frozen column divider/border.

Observed in current
current

Expected
expected

**Comment by alexjdetermined** [comment]
LGTM!

**Review by keita-determined — APPROVED** [review]
LGTM

File: webui/react/src/pages/F_ExpList/F_ExperimentList.tsx [hunk 1]
```diff
@@ -389,9 +391,29 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     updateSettings({ compare: !settings.compare });
   }, [settings.compare, updateSettings]);
 
+  const comparisonViewWidth = useMemo(() => {
+    return STATIC_COLUMNS.reduce(
+      (totalWidth, curCol) => totalWidth + settings.columnWidths[curCol] ?? 0,
+      17, // Constant of 17px accounts for scrollbar width
```
[the remark this answers is not in the document]
Comment (EmilyBonar) [reply webui/react/src/pages/F_ExpList/F_ExperimentList.tsx:405]: Not sure, but 17px was the minimum width that seemed to guarantee that there wouldn't be a horizontal scrollbar. I tested in Firefox and Chrome.

**Pull request merged**